### ISSUE TITLE
How to change date and time format?

### ISSUE THREAD

**Issue by Trameiro** [NoiSek/Aether#64]
I didn't get how to change it. I would like to change to pt_BR.

**Comment by NoiSek** [comment]
You can find the formatting options in the settings dialogue, "Date & Time" should be the very first thing you see. If you don't already know, you must hover over the bottom left of your screen to access the settings.

More information on how to format the date and time here:
https://github.com/samsonjs/strftime#supported-specifiers

**Comment by Trameiro** [comment]
> More information on how to format the date and time here:
> https://github.com/samsonjs/strftime#supported-specifiers

Yes but, can I change this?: https://github.com/samsonjs/strftime#bundled-locales

**Comment by NoiSek** [comment]
You'll have to manually specify the pt_BR datetime format using those formatting symbols, there's no support for changing it automatically by locale at the moment.

**Comment by Trameiro** [comment]
Where can I change it? Do I need any app?
